setup.py

```diff
@@ -66,7 +65,6 @@ def find_version(*file_paths):
         'sqlite': ["sqlalchemy"],
     },
     dependency_links=[
-        "git+https://github.com/ODM2/geoalchemy.git@v0.7.3#egg=geoalchemy-0.7.3",
         "git+https://github.com/ODM2/ODM2PythonAPI@v0.1.0-alpha#egg=odm2api-0.1.0"
```

Comment:
I don't know the answer to that. WOFpy does not seem to use `odm2api` directly in the code base.
I did see an example that relies on `odm2api` though.